test/jest/exportWebsocketMessages.js

```diff
@@ -0,0 +1,53 @@
+const fs = require("fs");
+
+function processHar(harPath) {
+  const jsonString = fs.readFileSync(harPath, { encoding: "utf8" });
+  const harContents = JSON.parse(jsonString);
+
+  const replayWebsocketEntry = harContents.log.entries.find(entry => {
+    return entry._resourceType === "websocket" && entry.request.url === "wss://dispatch.replay.io/";
+  });
+
+  if (!replayWebsocketEntry) {
+    console.error("Couldn't find a Replay websocket entry!");
+    process.exit(1);
+  }
+
+  const { _webSocketMessages: messages = [] } = replayWebsocketEntry;
+
+  console.log("Found replay entry. messages: ", messages.length);
+
+  const receivedMessages = messages.filter(message => {
+    if (message.type !== "receive") {
+      return false;
+    }
+
+    // Ignore messages that are empty or have large irrelevant contents
+    if (
+      message.data.includes('"result":{}') ||
+      message.data.includes("Network.requests") ||
+      message.data.includes("image/jpeg")
+    ) {
+      return false;
+    }
+
+    return true;
+  });
+
+  console.log("Matching messages: ", receivedMessages.length);
+
+  fs.writeFileSync("./websocketMessages.json", JSON.stringify(receivedMessages));
+}
+
+function main() {
+  const [node, script, harPath] = process.argv;
+
+  if (typeof harPath !== "string" || !harPath.endsWith(".har" || !fs.existsSync(harPath))) {
+    console.log("Must provide a valid path to a .har file");
+    process.exit(1);
+  }
+
+  processHar(harPath);
```

Comment:
I am pretty sure that we do not support network events in the console. which means we can probably remove this test and delete more code

Reply:
The `"Network.requests"` messages were definitely in the data sent via websocket from the backend.  They actually contained a bunch of request info.  Either that's being used by the Network Monitor, or the backend is sending them and they're not being used.

I stripped those out of the export to help shrink the size of the fixture file I committed, but also because I figured it was better to blanket remove those for now rather than try to do spend time combing through the exported websocket messages to make sure there wasn't any personal/sensitive info in the fixture.

Reply:
oh interesting - maybe they're shown in console warnings :/ good thing we're testing them :)